test/wpool_meta_SUITE.erl

```diff
@@ -9,7 +9,8 @@
 
 -export([all/0]).
 
--spec all() -> [dialyzer | elvis].
+%%-spec all() -> [dialyzer, elvis].
+-spec all() -> [atom()].
```

Comment:
I don't understand why you need to modify the spec this way when the function is clearly returning something with type `[dialyzer | elvis]`. Even more, it's returning something with type `[dialyzer | elvis, ...]` (i.e. the list can't be empty).